components/body/index.coffee

```diff
@@ -3,6 +3,15 @@ $ = require 'jquery'
 module.exports =
 
   initExpandContent: ->
-    $('body').on 'tap', '.body-content-container section', (e) ->
+    $('body').on 'tap', '.js-display-content', (e) ->
       e.preventDefault()
-      $(e.currentTarget).toggleClass 'is-active'
\ No newline at end of file
+
+      if $(e.currentTarget).hasClass('is-active')
+        $(e.currentTarget).removeClass('is-active')
+        $(e.currentTarget).siblings('.js-section-content').slideUp('slow')
+      else
+        $('.js-section-content').css('display', 'none')
+        $('.body-section > a').removeClass('is-active')
```

Comment:
Maybe a good follow up is to refactor the class structure so that we don't need to change individual element's class/css. But this is a good first step!